src/routes/patches/PatchItem.svelte

```diff
@@ -36,20 +37,48 @@
 			</a>
 		{/each}
 
-		<!-- should i hardcode this to get the version of the first package? idk you cant stop me -->
-		{#if patch.compatiblePackages.length && patch.compatiblePackages[0].versions.length}
-			<li class="patch-info">
-				🎯 {patch.compatiblePackages[0].versions.slice(-1)}
-			</li>
-		{/if}
-
 		{#if !patch.compatiblePackages.length}
 			<li class="patch-info">🌎 Universal patch</li>
 		{/if}
 
 		{#if hasPatchOptions}
 			<li class="patch-info">⚙️ Patch options</li>
 		{/if}
+
+		<!-- should i hardcode this to get the version of the first package? idk you cant stop me -->
+		{#if patch.compatiblePackages.length && patch.compatiblePackages[0].versions.length}
+			{#if showAllVersions}
+				{#each patch.compatiblePackages[0].versions
+					.slice()
+					.sort((a, b) => {
+						const coercedA = coerce(a);
+						const coercedB = coerce(b);
+						if (coercedA && coercedB) return compare(coercedA, coercedB);
+						else if (!coercedA && !coercedB) return 0;
+						else return !coercedA ? 1 : -1;
+					})
+					.reverse() as version}
+					<li class="patch-info">
+						🎯 {version}
+					</li>
+				{/each}
+			{:else}
+				<li class="patch-info">
+					🎯 {patch.compatiblePackages[0].versions.slice(-1)}
+				</li>
+			{/if}
+			{#if patch.compatiblePackages[0].versions.length > 1}
+				<li class="patch-info button" on:click={() => (showAllVersions = !showAllVersions)}>
+					<img
+						class="expand-arrow"
+						id="expand-versions"
+						style:transform={showAllVersions ? 'rotate(90deg)' : 'rotate(-90deg)'}
+						src="/icons/expand_more.svg"
+						alt="dropdown"
+					/>
+				</li>
```

Comment:
I suggest to move <img> out of <li>. The reason is that the semantics of the <li> here is that it is a chip, but the expand button is not a chip:

Before:
![image](https://github.com/ReVanced/revanced-website/assets/13122796/a28e40b6-f6c4-4d18-bc51-b05e84b5bf00)

After:
![image](https://github.com/ReVanced/revanced-website/assets/13122796/4064c445-b602-4f0d-811a-5027d1fe3ba2)